### PR TITLE
WRT-280: Fix info how to add features after release

### PR DESCRIPTION
**Pull request by anachuprina**
Preview: http://tradingview-rest-integrations-docs-staging.s3-website-us-east-1.amazonaws.com/branches/WRT-280-add-feature-after-release/Integration_overview.html#adding-features-after-release

### CODE REVIEW

File: source/Integration_overview.rst [hunk 1]
```diff
@@ -183,10 +183,13 @@ will have to:
...
-Adding features after the integration release
-................................................
-New features need to be added to the broker's staging environment and tested in the sandbox.
-The feature gets into production only after successful testing by the TradingView testing team.
+Adding features after release
+...............................
+
+If you want to add new features to the TradingView production environment after release,
+you first need to confirm the changes with the TradingView team.
+Once confirmed, you can implement the new feature on your :ref:`staging environment <trading-environments>`.
```
Comment (otroyko) [inline source/Integration_overview.rst:191]: Этот шаг обязательный. Тогда может быть не 'can' а 'should', или как-то перефразировать. Но тут надо у english-proofreading спрашивать.
Смысл в том, что без проверки на их стеджинге, как это сказано в 192 строке.

File: source/Integration_overview.rst [hunk 2]
```diff
@@ -183,10 +183,13 @@ will have to:
...
-................................................
-New features need to be added to the broker's staging environment and tested in the sandbox.
-The feature gets into production only after successful testing by the TradingView testing team.
+Adding features after release
+...............................
+
+If you want to add new features to the TradingView production environment after release,
+you first need to confirm the changes with the TradingView team.
+Once confirmed, you should implement the new feature on your :ref:`staging environment <trading-environments>`.
+Note that deploying new features to your production environment is only allowed after successful testing by the TradingView team.
```
Comment (KatPierce) [inline source/Integration_overview.rst:192]: This sentence is a bit too long and overwhelmed with gerund and passive voice. Let's try to make it simpler. 
```suggestion
Note that new features can be deployed to your production environment only after the TradingView team successfully test them.
```
or
```suggestion
Note that the TradingView team will deploy new features to your production environment only after successful testing.
```

File: source/Integration_overview.rst
Comment (KatPierce) [inline source/Integration_overview.rst:1]: > The sandbox is a fully functional copy of the TradingView website located at `beta-rest.xstaging.tv`_. Access to the
> resource is provided by adding an IP address to the whitelist on the TradingView side.

Addition to your changes. Can we clarify this paragraph a bit (L135)? Who adds an IP address to the whitelist? What should broker do to get access?

Comment (anachuprina) [reply source/Integration_overview.rst:1]: I've clarified this paragraph within another task https://jira.xtools.tv/browse/WRT-445. I will pass it to you soon c:

**Review by KatPierce — APPROVED** [review]

**Pull request merged**